### PR TITLE
Update Get-ADUser.md

### PR DESCRIPTION
**Pull request by doctordns**
1. Added an example using an LDAP Filter to get all enabled users.
2. Added PowerShell to the example's code fencing
3. Removed figure tense to improve readability
4. added a space between the prompt and the command in all examples

Addresses issue #2017

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 1]
```diff
@@ -118,6 +118,20 @@ Description
...
 Get the user with name 'GlenJohn' on the AD LDS instance.
 
+### -------------------------- EXAMPLE 5 --------------------------
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+Description
+
+-----------
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:128]:
```suggestion
```
Thank you @doctordns for this contribution!

Let's remove these lines in the examples.

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 2]
```diff
@@ -351,21 +365,21 @@ The following two examples show how to specify a value for this parameter.
...
 
 - If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.
 - If running cmdlets from an Active Directory provider drive, the default value of Partition is automatically generated from the current path in the drive.
-- If the target AD LDS instance has a default naming context, the default value of Partition will be set to the default naming context.  To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
-- If none of the previous cases apply, the Partition parameter will not take any default value.
+- If the target AD LDS instance has a default naming context, the default value of Partition is set to the default naming context.  To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
+- If none of the previous cases apply, the Partition parameter does not take any default value.
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:382]:
```suggestion
- If none of the previous cases applies, the Partition parameter does not take any default value.
```

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 3]
```diff
@@ -351,21 +365,21 @@ The following two examples show how to specify a value for this parameter.
 
 -Partition "CN=Schema,CN=Configuration,DC=EUROPE,DC=TEST,DC=CONTOSO,DC=COM"
 
-In many cases, a default value will be used for the Partition parameter if no value is specified. 
+In many cases, a default value is used for the Partition parameter if no value is specified. 
 The rules for determining the default value are given below. 
-Note that rules listed first are evaluated first and once a default value can be determined, no further rules will be evaluated.
+Note that rules listed first are evaluated first and once a default value can be determined, no further rules is evaluated.
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:370]:
```suggestion
Note that rules listed first are evaluated first, and once a default value can be determined, no further rules are evaluated.
```

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 4]
```diff
@@ -351,21 +365,21 @@ The following two examples show how to specify a value for this parameter.
...
 
-In AD DS environments, a default value for Partition will be set in the following cases:  - If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.
+In AD DS environments, a default value for Partition is set in the following cases:  - If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.
 
 - If running cmdlets from an Active Directory provider drive, the default value of Partition is automatically generated from the current path in the drive.
-- If none of the previous cases apply, the default value of Partition will be set to the default partition or naming context of the target domain.
+- If none of the previous cases apply, the default value of Partition is set to the default partition or naming context of the target domain.
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:375]:
```suggestion
- If none of the previous cases applies, the default value of Partition is set to the default partition or naming context of the target domain.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Thank you @doctordns for this contribution!

I think we should add the same example to 2012 R2 and 2016 versions.

**Review by get-itips — COMMENTED** [review]
I am ok once suggestions are applied.

**Comment by tseward** [comment]
Also approved once changes are implemented.

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 5]
```diff
@@ -351,21 +365,21 @@ The following two examples show how to specify a value for this parameter.
...
+In AD LDS environments, a default value for Partition is set in the following cases:
 
 - If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.
 - If running cmdlets from an Active Directory provider drive, the default value of Partition is automatically generated from the current path in the drive.
-- If the target AD LDS instance has a default naming context, the default value of Partition will be set to the default naming context.  To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
-- If none of the previous cases apply, the Partition parameter will not take any default value.
+- If the target AD LDS instance has a default naming context, the default value of Partition is set to the default naming context.  To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:381]:
```suggestion
- If the target AD LDS instance has a default naming context, the default value of Partition is set to the default naming context. To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
```

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 6]
```diff
@@ -118,6 +118,20 @@ Description
...
+### -------------------------- EXAMPLE 5 --------------------------
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+Description
+
+-----------
+
+Get all user accounts in AD that are enabled, using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:130]:
```suggestion
Get all user accounts in Active Directory that are enabled, using an LDAP filter.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Approved with changes implemented.

**Comment by e0i** [comment]
Please feel free to re-open this PR by implementing the changes suggested above, and resolving the merge conflicts and and the review is going to continue. Thank you.

**Pull request closed without merging**